runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterConfig.java

```diff
@@ -108,4 +109,37 @@ default boolean filterLogin()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "chatCountColor",
+		name = "Collapsed Count Color",
+		description = "Configures the color for the number of collapsed messages",
+		position = 8
+	)
+	default Color chatCountColor()
```

Comment:
IMO this seems kind of useless? If you just append the count it would take the color of whatever the message was, probably